components/TheHeader.vue

```diff
@@ -202,11 +214,29 @@ export default {
       return options.length ? options : null
     },
 
+    getLocaleOptions() {
+      const { $swell } = this
+
+      const options = $swell.locale
+        .list()
+        .map(locale => ({
+          value: locale.code,
+          label: locale.name,
+          icon: locale.icon
+        }))
+      return options.length ? options : null
+    },
+
     selectCurrency(value) {
       const { $swell } = this
       this.$store.dispatch('selectCurrency', { code: value })
     },
 
+    selectLocale(value) {
+      const { $swell } = this
+      this.$store.dispatch('selectLocale', { code: value })
+    },
+
```

Comment:
@ericingram Any reason we're using `value` as the param instead of `code` here?

Reply:
Not that I can think of, i'd go with code

Reply:
With my commit 0634d9a, I've had to change it to be `{ code: options.value }`, as I've refactored the logic to fit within the respective component.